_posts/patterns/2015-04-15-icons.md

```diff
@@ -877,7 +899,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Quote
+            quote-left
```

Comment:
I wondered about this too. I'm not adding any new icons as part of this change, but we should consider if this is necessary in a future PR.